prowler/providers/aws/services/ec2/ec2_service.py

```diff
@@ -38,6 +38,10 @@ def __init__(self, provider):
         self.__threading_call__(self.__get_ebs_encryption_settings__)
         self.elastic_ips = []
         self.__threading_call__(self.__describe_ec2_addresses__)
+        self.ebs_block_public_access_snapshots_state = []
+        self.__threading_call__(self.__get_snapshot_block_public_access_state__)
+        self.instance_metadata_defaults = []
```

Comment:
If you include this into the Snapshot class this could be removed.

Reply:
N/A

Reply:
N/A